thirdparty/nlohmann_json/nlohmann_json/src/single_include/nlohmann_json.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef __NLOHMANN_JSON_HPP__
```

Comment:
@chuckatkins I added this wrapper for the nlohmann_json library in order to be able to disable warnings in the library